plugins/azure/postgresqlserver/flexibleServerLogDisconnections.js

```diff
@@ -0,0 +1,64 @@
+const async = require('async');
+const helpers = require('../../../helpers/azure');
+
+module.exports = {
+    title: 'Postgresql FLexible Server Log Disconnections Enabled',
+    category: 'PostgreSQL Server',
+    domain: 'Databases',
+    severity: 'Medium',
+    description: 'Ensures disconnection logs are enabled  for Postgresql flexible servers',
+    more_info: 'Disconnection logs ensure all attempted and successful disconnections from the server are logged.',
+    recommended_action: 'Ensure the server parameters for each Postgresql flexible server have the log_disconnections setting enabled.',
+    link: 'https://learn.microsoft.com/en-us/azure/postgresql/flexible-server/how-to-configure-server-parameters-using-portal',
+    apis: ['servers:listPostgresFlexibleServer', 'flexibleServersConfigurations:listByPostgresServer'],
+    realtime_triggers: ['microsoftdbforpostgresql:flexibleservers:write','microsoftdbforpostgresql:flexibleservers:delete','microsoftdbforpostgresql:flexibleservers:configurations:write'],
+
+    run: function(cache, settings, callback) {
+        const results = [];
+        const source = {};
+        const locations = helpers.locations(settings.govcloud);
+
+        async.each(locations.servers, (location, rcb) => {
+            const servers = helpers.addSource(cache, source,
+                ['servers', 'listPostgresFlexibleServer', location]);
+            if (!servers) return rcb();
+
+            if (servers.err || !servers.data) {
+                helpers.addResult(results, 3,
+                    'Unable to query for PostgreSQL flexible servers: ' + helpers.addError(servers), location);
+                return rcb();
+            }
+        
+            if (!servers.data.length) {
+                helpers.addResult(results, 0, 'No existing PostgreSQL flexible servers found', location);
+                return rcb();
+            }
+
+            for (var flexibleServer of servers.data) {
+                const configurations = helpers.addSource(cache, source,
+                    ['flexibleServersConfigurations', 'listByPostgresServer', location, flexibleServer.id]);
+        
+                if (!configurations || configurations.err || !configurations.data) {
+                    helpers.addResult(results, 3,
+                        'Unable to query for configuration' + helpers.addError(configurations), location, flexibleServer.id);
```

Comment:
i think we should add the message like Unable to query for PostgreSQL flexible server configurations